src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.PicturesToNumber.Controllers;
+
+
+import io.swagger.annotations.ApiParam;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import java.sql.SQLException;
+
+@RestController
+@RequestMapping("/api")
+public class PictureController {
+
+   @RequestMapping(value = "/fileUpload", method = RequestMethod.POST)
+    public @ResponseBody String uploadFile(@RequestPart("file") MultipartFile file, HttpServletRequest req)
+            throws SQLException {
```

Comment:
Why do you need SqlException?

Reply:
Sorry, we really don't need it, I've already changed it